### ISSUE TITLE
undefined name 'out'

### ISSUE THREAD

**Issue by williballenthin** [mandiant/speakeasy#197]
@dtrizna can you address this undefined name exception introduced here: https://github.com/mandiant/speakeasy/commit/208307abcd0e4255f0df4c537d8f4c92ab65d413

```
./speakeasy/winenv/api/usermode/kernel32.py:5747:44: F821 undefined name 'out'
            self.mem_write(lpConsoleTitle, out)
                                           ^
1     F821 undefined name 'out'
1
```

https://github.com/mandiant/speakeasy/blob/fa28af418208890e5b2b0d4da459ab3240f8c486/speakeasy/winenv/api/usermode/kernel32.py#L5747

**Comment by williballenthin** [comment]
closed in #198

**Comment by williballenthin** [comment]
thanks @dtrizna!
